src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+import java.util.Scanner;
+import java.lang.String;
+
 public class Duke {
+    static final int MAX_NO = 100;
+    private static int numOfTasks = 0; // stores the number of tasks in the array
+    private static Task[] tasks = new Task[MAX_NO]; // initialise array of Task objects
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printGreeting();
+        getMessage();
+    }
+
+    /**
+     * gets command(s) from the user and executes it(them) appropriately
+     * available commands: list, done, deadline, event, todo, task, bye
+     */
+    private static void getMessage() {
+        Scanner in = new Scanner(System.in);
+        String[] input = new String[MAX_NO];
+        int i = 0;
+
+        while (true) {
+            input[i] = in.nextLine();
+            if (input[i].equals("bye")) {
+                printBye();
+                break;
+            } else if (input[i].equals("list")) {
+                listTasks(tasks);
+            } else if (input[i].contains("done")) {
+                markAsDone(input[i]);
+            } else if (input[i].contains("deadline")){
+                addDeadline(input[i]);
+            } else if (input[i].contains("event")) {
+                addEvent(input[i]);
+            } else if (input[i].contains("todo")) {
+                addToDo(input[i]);
+            } else {
+                echoMessage(input[i]);
+            }
+            i++;
+        }
+    }
+
+    public static void printGreeting() {
+        String logo = "      /-\\    /-\\ \n" //6 spaces
+            + "     /  |_9_/  |\n" //5 spaces
+            + "    /,,o  3  o,,\\ \n"; //4 spaces
+        printDivider();
+        System.out.println("\tHullo I'm Toto!\n" + logo); //4 spaces
+        System.out.println("\tHow can Toto help today?"); //4 spaces
+        printDivider();
+    }
+
+    public static void printDivider(){
+        System.out.println("-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-");
+    }
+
+    public static void printBye(){
+        String logo = "      /-\\    /-\\ \n"
+            + "     /  |_9_/  |\n"
+            + "    / TT  w  TT \\ \n";
+        printDivider();
+        System.out.println("\tToto will be lonely... :<\n" + logo);
+        printDivider();
+    }
+
+    public static void listTasks (Task[] tasks) {
+        printDivider();
+        if (numOfTasks == 0) {
+            System.out.println("\tyour task list is empty");
+            printDivider();
+        } else {
+            for (int j = 0; j < numOfTasks; j++) {
+            System.out.println((j+1) + ": " + tasks[j]);
+            }
+        }
+        printDivider();
+    }
+
+    public static void addDeadline(String input) {
+        numOfTasks++;
```

Comment:
May be better to spell out number, similarly for the other variables that has "num"